lib/Doctrine/ORM/Cache/CacheConfiguration.php

```diff
@@ -1,5 +1,8 @@
 <?php
 
+declare(strict_types = 1);
```

Comment:
@Majkl578 we can land it if it passes with green and not too many test changes. We'll need to look at the changed tests and see whether there are serious BC breaks for consumers.